xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/internal/html/DefaultHTMLCleanerTest.java

```diff
@@ -661,18 +664,104 @@ public void parseWithUnicodeChars() throws Exception
 
         assertEquals("baz&buz", nodeList.item(2).getAttributes().getNamedItem("foo").getTextContent());
 
-        // BUG: This is failing with baz&gt;buz
+        // This was failing with baz&gt;buz
```

Comment:
Why keep the comment if it's passing now?

Reply:
Maybe I wanted to keep a note why we actually test this here. Then, if it is confusing, I am happy to scrap the comment.